arlo-client/src/components/Audit/index.tsx

```diff
@@ -79,8 +80,13 @@ const Audit: React.FC<{}> = () => {
   const [menuItems, refresh] = useSetupMenuItems(stage, setStage, electionId)
 
   useEffect(() => {
-    refresh()
-  }, [refresh])
+    if (
+      isAuthenticated &&
+      viewMatch === 'setup' &&
+      meta!.type === 'audit_admin'
+    )
+      refresh()
+  }, [refresh, isAuthenticated, viewMatch, meta])
```

Comment:
This is actually related to your other comment about restructuring so that the multi jurisdiction and single page flows are completely separated. That solution would make so I don't have to do this, haha. Basically, this makes it so that it won't try to update the AA setup menu items when you won't be using them (and also won't have the authorization to do so). Otherwise it will attempt to access the endpoints that are only allowed to audit admins, and it throws errors. This is just a stopgap until we can refactor.